<?xml version="1.0"?>
<launch>

  <arg name="sim" default="true" />

  <group if="$(arg sim)">
    <node name="servo_server" pkg="moveit_servo" type="servo_server" output="screen" >
      <param name="parameter_ns" type="string" value="optional_parameter_namespace" />
      <rosparam ns="optional_parameter_namespace" command="load" file="$(find moveit_pose_tracking)/config/robot_config_sim.yaml" />
    </node>
  </group> 

  <group unless="$(arg sim)">
    <node name="servo_server" pkg="moveit_servo" type="servo_server" output="screen" >
      <param name="parameter_ns" type="string" value="optional_parameter_namespace" />
      <rosparam ns="optional_parameter_namespace" command="load" file="$(find moveit_pose_tracking)/config/robot_config.yaml" />
    </node>
  </group> 

</launch>
